cbmc/proofs/IotMqtt_SubscribeAsync/Makefile

```diff
@@ -0,0 +1,68 @@
+ENTRY=IotMqtt_SubscribeAsync
+
+# We abstract all unreachable functions to improve coverage metrics
+ABSTRACTIONS += --remove-function-body _checkRetryLimit
+ABSTRACTIONS += --remove-function-body _IotMqtt_PublishSetDup
+ABSTRACTIONS += --remove-function-body _IotMqtt_SerializeUnsubscribeCommon
+ABSTRACTIONS += --remove-function-body _IotMqtt_SerializeUnsubscribe
+ABSTRACTIONS += --remove-function-body _mqttOperation_match
+ABSTRACTIONS += --remove-function-body _mqttSubscription_setUnsubscribe
+ABSTRACTIONS += --remove-function-body _packetMatch
+ABSTRACTIONS += --remove-function-body _scheduleNextRetry
+ABSTRACTIONS += --remove-function-body _topicFilterMatch
+ABSTRACTIONS += --remove-function-body IotNetworkInterfaceClose
+ABSTRACTIONS += --remove-function-body IotNetworkInterfaceReceive
+
+OBJS += $(ENTRY)_harness.goto
+OBJS += $(MQTT)/cbmc/proofs/mqtt_state.goto
+OBJS += $(MQTT)/libraries/standard/mqtt/src/iot_mqtt_api.goto
+OBJS += $(MQTT)/libraries/standard/mqtt/src/iot_mqtt_helper.goto
+OBJS += $(MQTT)/libraries/standard/mqtt/src/iot_mqtt_operation.goto
+OBJS += $(MQTT)/libraries/standard/mqtt/src/iot_mqtt_serialize.goto
+OBJS += $(MQTT)/libraries/standard/mqtt/src/iot_mqtt_subscription.goto
+OBJS += $(MQTT)/libraries/standard/mqtt/src/iot_mqtt_validate.goto
+
+# One more than actual number of subscriptions in a subscription list
+SUBSCRIPTION_COUNT_MAX=2
+DEF += -DSUBSCRIPTION_COUNT_MAX=$(SUBSCRIPTION_COUNT_MAX)
+
+# One more than actual number of operations in an operation list
+OPERATION_COUNT_MAX=2
+DEF += -DOPERATION_COUNT_MAX=$(OPERATION_COUNT_MAX)
+
+# One more than actual length of topics
+TOPIC_LENGTH_MAX=3
```

Comment:
Does this need to be so small?  Even 6 or 10 would be more comfortable.

Reply:
I increased the size to 6, which increased the run-time to 26m8.718s. I tried with 10, but the run-time was 39m37.392s.